Cesium.Compiler/Properties/launchSettings.json

```diff
@@ -0,0 +1,20 @@
+{
+    "profiles": {
+        "Cesium.Compiler": {
+            "commandName": "Project",
+            "commandLineArgs": "C:\\d\\github\\Cesium\\Cesium.IntegrationTests\\stdlib/iso646.c -o C:\\d\\github\\Cesium\\Cesium.IntegrationTests/bin/out_cs.exe"
```

Comment:
That's a hardcode I'd like us to avoid. I have nothing against adding some kind of test launchsettings file, though. Maybe let's generalize this a bit and maybe point to our samples directory?

Reply:
I have to take a look how to do that properly. For now I just remove that file. anyway I add it by mistake.